crypto/pem/pem_lib.c

```diff
@@ -77,13 +77,6 @@ int pem_check_suffix(const char *pem_str, const char *suffix);
 
 int PEM_def_callback(char *buf, int num, int w, void *key)
 {
-#if defined(OPENSSL_NO_STDIO) || defined(OPENSSL_NO_UI)
-    /*
-     * We should not ever call the default callback routine from windows.
-     */
-    PEMerr(PEM_F_PEM_DEF_CALLBACK, ERR_R_SHOULD_NOT_HAVE_BEEN_CALLED);
-    return (-1);
-#else
     int i, j;
     const char *prompt;
```

Comment:
Compiling with the `--strict-warnings` config option reveals that `j` and `prompt` are unused in `no-ui` and `no-stdio` case.